resources.qrc

```diff
@@ -0,0 +1,5 @@
+<RCC>
```

Comment:
I'm not sure `rc` directory is for Windows Resource Files only or not so I put `resources.qrc` outside.